core/util/proxy.py

```diff
@@ -0,0 +1,108 @@
+import requests
+from bs4 import BeautifulSoup
+import time
+
+class main:
+
+    def __init__(self, use=False, verbose=False):
+        '''
+        use : Determine whether to use proxy or not.
+        '''
+        self.use=use
+        self.verbose=verbose
+    iplist=[]
```

Comment:
not here

Reply:
Sir I didn't quite get what you meant. 😅

Should I remove `iplist=[]` from here? If yes, where and how should I write it? As `self.iplist=[]`?